common/src/main/java/bisq/common/util/NetworkUtils.java

```diff
@@ -137,6 +140,11 @@ public static Optional<String> findLANHostAddress(Optional<NetworkInterface> pre
         }
         return networkInterfaceHostAddressPairs.stream()
                 .map(Pair::getSecond)
-                .findFirst();
+                .distinct()
```

Comment:
log message 
`log.warn("No networkInterface found which matches the preferredNetworkInterface. We return all LAN addresses.");` was wrong already before the change. Can you update it with?

`log.warn("Preferred network interface not found. Falling back to auto-selected LAN address.");`